src/Sidenav/Treeview.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useCallback } from 'react';
+import { RsRefForwardingComponent } from '../@types/common';
+import useTreeControl from './useTreeControl';
+import TreeControlContext from './TreeControlContext';
+import useEnsuredRef from '../utils/useEnsuredRef';
+
+const Treeview: RsRefForwardingComponent<
+  'ul',
+  React.HTMLAttributes<HTMLUListElement>
+> = React.forwardRef<HTMLUListElement, React.HTMLAttributes<HTMLUListElement>>((props, ref) => {
+  const treeControl = useTreeControl();
+  const treeRef = useEnsuredRef<HTMLUListElement>(ref);
+
+  const handleFocus = useCallback(
+    (event: React.FocusEvent) => {
+      if (event.target === treeRef.current) {
+        treeControl.handleReceiveFocus(event);
+      }
+    },
+    [treeControl]
```

Comment:
React Hook useCallback has a missing dependency: 'treeRef'. Either include it or remove the dependency array